packages/sdk/src/contracts/factory.contract.ts

```diff
@@ -220,4 +241,38 @@ export class Factory extends HolographBaseContract {
   ) {
     return this._getContractFunction({chainId, wallet, functionName: 'bridgeOut', args: [toChain, sender, payload]})
   }
+
+  /**
+   * @readonly
+   * Get the current gas price for a certain network.
+   * @param chainId The chain id of the network to get the result from.
+   * @returns The gas price in wei.
+   */
+  async getGasPrice(chainId: number) {
+    return this._getGasPrice(chainId)
+  }
+
+  /**
+   *
+   * @param chainId  The chain id of the network to send the transaction.
+   * @param config The contract deployment configurations.
+   * @param signature The signature by the wallet that created the original payload.
+   * @param signer The address of the signer.
+   * @param wallet Holograph wallet instance, optional param.
+   * @returns The estimated gas for the transaction in wei.
+   */
+  async estimateGasForDeployingHolographableContract(
```

Comment:
@feliperodriguess What is the reason for putting this function in the holograph factory class? My only guess is that this estimate requires uses the factory contract.

Reply:
Yes, it requires, and it's tied to the deployHolographableContract, which is a function from the factory contract

Reply:
The above is just a wrapper, the final version lives on collection-legacy.ts as you can see on this PR (https://github.com/holographxyz/holograph/pull/32)

Reply:
ok got it.